assembl/static2/js/app/components/common/avatar.jsx

```diff
@@ -1,9 +1,10 @@
 import React from 'react';
+import { customBrowserHistory } from '../../index';
```

Comment:
customBrowserHistory is not currently exported in index.jsx. I don't see the change in your PR. You need to add `export customBrowserHistory` in index.jsx.

Reply:
That's right... why didn't webpack complain? (I even tested!)

Reply:
webpack will not complain, eslint should have. Try `npm run eslint`

Reply:
ok, that was helpful, thank you. Have a look now?

Reply:
Ok, should be good.